src/config.js

```diff
@@ -20,6 +21,12 @@ const MAPPED_STATUS_COMMANDS = {
   'pr-merged-in-dev': '✅',
 };
 
+const PR_TEMPLATE = {
+  allowedChannel: parseAllowedChannels(
```

Comment:
Maybe the name of the key would be in plural?

Reply:
Completed